interface/main/tabs/menu/menus/standard.json

```diff
@@ -724,7 +724,7 @@
     "menu_id": "admimg",
```

Comment:
also need to copy this file (with the change) to sites/default/documents/custom_menus/Custom.json (this is basically always the same file as interface/main/tabs/menu/menus/standard.json)